### PR TITLE
KAFKA-6420: Adding Msys support

### PR DESCRIPTION
**Pull request by asutosh936**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: bin/kafka-run-class.sh [hunk 1]
```diff
@@ -20,8 +20,8 @@ then
   exit 1
 fi
 
-# CYGINW == 1 if Cygwin is detected, else 0.
-if [[ $(uname -a) =~ "CYGWIN" ]]; then
+# CYGINW == 1 if Cygwin is detected or Msys is detected, else 0.
```
Comment (kamalcph) [inline bin/kafka-run-class.sh]: nit: Also, update the comment `CYGINW` -> `CYGWIN`

Comment (asutosh936) [reply bin/kafka-run-class.sh]: Done

**Comment by harshach** [comment]
+1

**Comment by asutosh936** [comment]
@guozhangwang : Could You or someone from Your team review and provide feedback on this PR?

**Comment by vvcephei** [comment]
Hey @asutosh936 , Can you provide a description in the PR? It would become the commit message, and it will also help give reviewers context.

Thanks,
-John

**Comment by XT3RM1N8R** [comment]
It has been almost 3 years.
Would it be bad form if I were to submit my own very similar PR to revive this effort?
I would like to help.

**Pull request closed without merging**